test/controllers/name_controller_test.rb

```diff
@@ -2751,6 +2752,41 @@ def test_update_name_reverse_merge_add_identifier
     assert_equal(189_826, edited_name.reload.icn_id)
   end
 
+  def test_update_merge_destroy_approved_synonym
+    approved_synonym = names(:lactarius_alpinus)
+    deprecated_name = names(:lactarius_alpigenes)
+    Naming.create(user: mary,
+                  name: deprecated_name,
+                  observation: observations(:minimal_unknown_obs))
+    assert(
+      !approved_synonym.deprecated &&
+        Naming.where(name: approved_synonym).none? &&
+        deprecated_name.synonym == approved_synonym.synonym,
+      "Test needs different fixture: " \
+      "an Approved Name without Namings, with a synonym having Naming(s)"
```

Comment:
Should I replace this with a comment?
I'm not in love with hitting the db in a test simply to check whether someone modified the fixture in a way that breaks this test.

Reply:
I've done this in a few places, too.  Makes sense to me, especially in a case like this where it is reasonable to anticipate that a fixture really might change.  Probably not a big performance hit.